Dockerfile

```diff
@@ -8,11 +8,16 @@ RUN mkdir -p /src/project
 COPY project/ /src/project
```

Comment:
You don't need to copy the source into the docker container if you're installing from pip